jcalapi/app.py

```diff
@@ -25,6 +25,20 @@
 CACHE_EXPIRY = 60 * 10  # 10 minutes
 CACHE_RESTORED = ContextVar("CACHE_RESTORED", default=False)
 
+PAST_DAYS_IMPORT = int(os.environ.get("PAST_DAYS_IMPORT", 0))
+START_DATE = (
+    (
+        datetime.datetime.now(tz=tzlocal()).replace(
+            hour=0, minute=0, second=0, microsecond=0
+        )
+        - datetime.timedelta(days=PAST_DAYS_IMPORT)
+    )
+    if PAST_DAYS_IMPORT > 0
+    else None
+)
+END_DATE = datetime.datetime.now(tz=tzlocal()).replace(
+    hour=0, minute=0, second=0, microsecond=0
+) + datetime.timedelta(days=14)
```

Comment:
This probably shouldn't be hardcoded here.
Would make sense to have a `FUTURE_DAYS_IMPORT` that may default to 14

Reply:
It's possible of course. But i've based this on your code :
https://github.com/pschmitt/jcalapi/blob/4f4293e4e8e6a11937bdf748630e3a2036eaa532/jcalapi/backend/exchange.py#L149

Do you want i push a modification including this ?

Reply:
Fair point.

Rephrasing: I should have made a var for that instead of hardcoding 14 days.

If you're up for implementing this that'd be sweet. Otherwise we can always do this outside of this PR.